presto-main/src/main/java/com/facebook/presto/spiller/LocalSpillContext.java

```diff
@@ -15,7 +15,10 @@
 
 import com.facebook.presto.operator.AbstractOperatorSpillContext;
 
+import java.io.Closeable;
```

Comment:
yes, there is one commit that you didn't address before that.

Maybe you pushed them to github in wrong order?